chart/templates/arkime-ini.yml

```diff
@@ -6,7 +6,7 @@ data:
   config.ini: |
```

Comment:
There were a few [changes](https://github.com/idaholab/Malcolm/pull/412/files#diff-8fbcd5058dbfc41ac27f41fd47048639404fb707dcfa5a604305e52bf321346a) to config.ini, and you've got most of them. I do see we are missing `footerTemplate`, which defines the footer links at the bottom of the arkime UI (see ). Not a huge deal, it won't break anything, but you won't have the Malcolm URLs at the bottom.

I see you've got `spiViewCategoryOrder` as well already which is good (this just makes it so the "Malcolm" category gets bumped to the top in the SPIView). as well as the field definitions, so that's all good.

Reply:
Added the missing footer